ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/ShowPopup/ShowPopup.qml

```diff
@@ -47,7 +48,7 @@ Item {
         id: model
         mapView: view
 
-        onPopupManagersChanged: popupStackView.show();
```

Comment:
For the sake of portability, I think we should consider adding a show method if it is not present in the current beta_update10 branch. It seems pretty harmless and gets rid of actual code changes

Reply:
If we never got feedback from beta users on this, I would be tempted to say keep as is (but I do also have a personal investment in keeping the interface as pure/minimal as possible.) 

It is a minor addition so my arm can be twisted!

Reply:
Let's leave it off and add it if we need it